.travis.yml

```diff
@@ -8,8 +8,6 @@ python:
     - pypy3
 matrix:
     include:
-        - python: "2.7"
-          env: TOXENV=flake8,docs
```

Comment:
It makes me sad that we no longer run flake8/docs on every PR.

I don't feel strong enough about that to veto this change, but I'd like somebody else to speak up and give their opinion.  @icemac, you originally added these, what do you think?